settings.py

```diff
@@ -860,26 +860,26 @@
 # The variables should be used while reindexing an index.
 # When the variables are set to true the data will be written to both primary and secondary indexes.
 
-ES_APPS_INDEX_MULTIPLEXED = True
-ES_CASE_SEARCH_INDEX_MULTIPLEXED = True
-ES_CASES_INDEX_MULTIPLEXED = True
-ES_DOMAINS_INDEX_MULTIPLEXED = True
-ES_FORMS_INDEX_MULTIPLEXED = True
-ES_GROUPS_INDEX_MULTIPLEXED = True
-ES_SMS_INDEX_MULTIPLEXED = True
-ES_USERS_INDEX_MULTIPLEXED = True
+ES_APPS_INDEX_MULTIPLEXED = False
```

Comment:
@millerdev I feel we would be in a good place to swap the settings in this PR itself as the migration would verify that indices are in consistent state. And if indices are good we can swap the indices and release the ES upgrade changelog after that.

How do you feel about it?

Reply:
This puts the environment in a state where "It is unsafe to turn on multiplexing again" since "the secondary index has been promoted to primary and multiplexing is disabled."

Imagine a scenario where this code is deployed and one or more of the migration checks fail. At that point is it possible to start a celery or webworker process before resolving the migration failure? If yes, I think this means it is unsafe to revert to `ES_WHATEVER_INDEX_MULTIPLEXED = True`. Is there any reason someone would want to do that?

Reply:
> this code is deployed and one or more of the migration checks fail.

I might have a flawed understanding of it, but If the checks will fail, will this code be deployed? We are adding this migration just to ensure that this sort of scenario does not happen.
In my unde

> At that point is it possible to start a celery or webworker process before resolving the migration failure?

I think these settings will not come into effect until the symlink is updated, right? And if celery is started again when the migration fails the system would be using previous release which has previous settings.

Reply:
> I think these settings will not come into effect until the symlink is updated, right? 

Yes, you are correct. It slipped my mind that migrations are run before the symlink is updated. So the answer to my question

> is it possible to start a celery or webworker process before resolving the migration failure?

Is "No." This is good.